src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -198,6 +198,7 @@ protected void LookupMembersInType(LookupResult result, TypeSymbol type, string
                     break;
 
                 case TypeKind.Pointer:
+                case TypeKind.FunctionPointer:
```

Comment:
>case TypeKind.FunctionPointer: [](start = 16, length = 30)

Is this code path covered by tests? I didn't see any obviously related new test.

Reply:
Yes, this codepath was exposed by invoking an extension method on a function pointer type.

---
In reply to: [408290828](https://github.com/dotnet/roslyn/pull/43248#discussion_r408290828) [](ancestors = 408290828)

Reply:
Would we hit it for a simple attempt to call ToString or GetHashCode of a function pointer? Consider adding a test like that then. Also, it looks like you missed this usage of TypeKind.Pointer during the sweep. Consider taking another pass to make sure nothing is skipped unintentionally.

---
In reply to: [408295464](https://github.com/dotnet/roslyn/pull/43248#discussion_r408295464) [](ancestors = 408295464,408290828)

Reply:
Added a test. I looked through the rest, the only remaining things are nullable or public API related.

---
In reply to: [408303218](https://github.com/dotnet/roslyn/pull/43248#discussion_r408303218) [](ancestors = 408303218,408295464,408290828)